juno/assetlist.json

```diff
@@ -1849,6 +1849,33 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/ampjuno.svg"
         }
       ]
-    }
+    },
+    {
+      "description": "BITS - Native token of Bitswift Cash",
+      "type_asset": "cw20",
+      "address": "juno14fz92ehqt37e096xr95kmy8nc0kz803uezxtg4fwx7agjjma86sqm8mg3h",
+      "denom_units": [
+        {
+          "denom": "cw20:juno14fz92ehqt37e096xr95kmy8nc0kz803uezxtg4fwx7agjjma86sqm8mg3h",
+          "exponent": 0
+        },
+        {
+          "denom": "BITS",
+          "exponent": 8
+        }
+      ],
+      "base": "cw20:juno14fz92ehqt37e096xr95kmy8nc0kz803uezxtg4fwx7agjjma86sqm8mg3h",
+      "name": "BITS",
+      "display": "bits",
+      "symbol": "BITS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/bits.png"
```

Comment:
I'm a little concerned about the low fidelity of the image here. Do we have a version that looks cleaner? There' compression artifacts, blurry edges, no transparency layer, and some random dot in the top right corner...